src/client/sandbox/xhr.js

```diff
@@ -1,11 +1,13 @@
 import SandboxBase from './base';
 import nativeMethods from './native-methods';
-import { getProxyUrl } from '../utils/url';
+import { getProxyUrl, parseProxyUrl } from '../utils/url';
 import XHR_HEADERS from '../../request-pipeline/xhr/headers';
 import AUTHORIZATION from '../../request-pipeline/xhr/authorization';
 import { getOriginHeader } from '../utils/destination-location';
 import reEscape from '../../utils/regexp-escape';
 import * as JSON from '../json';
+import overrideDescriptor from '../utils/override-descriptor';
+import { SAME_ORIGIN_CHECK_FAILED_STATUS_CODE } from '../../request-pipeline/xhr/same-origin-policy';
 
 const IS_OPENED_XHR               = 'hammerhead|xhr|is-opened-xhr';
```

Comment:
Do we still need `XhrSandbox.isOpenedXhr` and `domUtils.isXMLHttpRequest` functions?